### PR TITLE
Fixed zstd for hermetic python toolchain

### PR DESCRIPTION
**Pull request by UebelAndre**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (please, look at the "Scope of the project" section in the README.md file)
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Another batch of suggestions for https://github.com/bazelbuild/rules_python/pull/618

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: python/repositories.bzl [hunk 1]
```diff
@@ -224,8 +235,6 @@ def python_register_toolchains(name, python_version, **kwargs):
 
     for platform in PLATFORMS.keys():
         sha256 = TOOL_VERSIONS[python_version]["sha256"].get(platform, None)
-        if not sha256:
```
Comment (f0rmiga) [inline python/repositories.bzl:227]: Also need to revert this.

Comment (UebelAndre) [reply python/repositories.bzl:227]: I don't think it should be reverted. `python_repository` should allow `sha256` to be optional. Which is common behavior of the `http_archive` and `http_file` rules.

Comment (UebelAndre) [reply python/repositories.bzl:227]: If anything I could see this being updated to a `fail` message but this skip leads to a confusing user experience IMO.

Comment (f0rmiga) [reply python/repositories.bzl:227]: Without this, it will break on versions that don't have a specific platform available.

**Comment by f0rmiga** [comment]
Thanks. I reverted the needed bits in d61f2f7.

**Pull request closed without merging**